project/FreeGen2.scala

```diff
@@ -521,7 +603,8 @@ class FreeGen2(managed: List[Class[_]], pkg: String, renames: Map[Class[_], Stri
     val ki = {
       val file = new File(base, s"kleisliinterpreter.scala")
       val pw = new PrintWriter(file)
-      pw.println(kleisliInterpreter)
+      if (base.getPath().contains("postgres")) pw.println(kleisliInterpreter)
+      else pw.println(kleisliInterpreterCancelable)
```

Comment:
What's the specific check here for?

Reply:
`doobie.postgres.free.KleisliInterpreter` doesn't need cancel feature at all, so we can generate ordinary interpreter.